torch_xla/distributed/xla_dist.py

```diff
@@ -40,6 +43,21 @@ class DistributedExecutor(object):
       'XRT_SHARD_ORDINAL',
   ]
   DEFAULT_CONTAINER_NAME = 'pytorchtpudistrunner'
+  MAX_TPU_RETRY = 50
+
+  @classmethod
+  def _get_logger(cls):
```

Comment:
No need to make it a class method.
Below you also call it with `self.` which makes this a bit confusing.

Reply:
Done.